back/app/services/events/side_fx_attendance_service.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Events
+  class SideFxAttendanceService < BaseSideFxService
```

Comment:
Wasn't aware of BaseSideFxService, not a big fan tbh, but that's another discussion. Added to CDM agenda.

Reply:
Can you elaborate, or you prefer keeping this for the CDM? :)